ONSdigital/dp-frontend-filter-dataset-controller
# Feature/1750 migrate app to dp renderer

### What

- ✅ [Convert app to use dp-renderer](https://trello.com/c/40dibcN7/1750-migrate-dp-frontend-filter-dataset-controller-to-use-dp-renderer)
- Enabled linter and implemented suggestions

### How to review

- Sense check
- All CI tasks pass
- Templates/models are lift and shift from `dp-frontend-renderer` and `dp-frontend-models`

Have a go locally
- Pull this branch `make debug`
- Port forward onto sandbox web api router
- Run `sixteens`
- Check a cmd filter record

### Who can review

!me
